src/algebra/module/localized_module.lean

```diff
@@ -999,6 +1004,80 @@ end
 
 end algebra
 
-end is_localized_module
+variables [module (localization S) M'] [is_scalar_tower R (localization S) M']
+/--
+- If `(f : M →ₗ[R] M')` is a localization of modules, then the map
```

Comment:
I think the mathlib style would be not to have those `-` signs at the beginning of every line in the module docstring (they will I suspect display in a pretty weird way in lots of situations).